playbooks/roles/jenkins_master/defaults/main.yml

```
@@ -8,7 +8,7 @@ jenkins_protocol_https: true
 jenkins_job_venv_dir: "/edx/var/jenkins/jobvenvs/"
 
 JENKINS_VERSION: '1.651.3'
-jenkins_deb_url: "https://pkg.jenkins.io/debian-stable/binary/jenkins_{{ JENKINS_VERSION }}_all.deb"
+jenkins_deb_url: "http://mirrors.jenkins-ci.org/debian-stable/jenkins_{{ JENKINS_VERSION }}_all.deb"
```

```suggestion
jenkins_deb_url: "https://archives.jenkins-ci.org/debian-stable/jenkins_{{ JENKINS_VERSION }}_all.deb"
```
It seems mirrors.jenkins-ci.org doesn't have a certificate in that name and we should keep using TLS. The old URL redirects to archices.jenins-ci.org so we should probably use that instead.

---

Oo thank you for that @jdmulloy !